sql/src/main/java/org/apache/druid/sql/calcite/rule/DruidExtensionCalciteRuleManager.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.rule;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import org.apache.calcite.plan.RelOptRule;
+import org.apache.druid.sql.calcite.planner.PlannerContext;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Manages the custom calcite rules coming from extensions
+ */
+public class DruidExtensionCalciteRuleManager
```

Comment:
naming nit: no real need to have the word "druid" in here. ExtensionCalciteRuleManager is clear enough.